lib/aws/session_store/dynamo_db/version.rb

```diff
@@ -1,21 +1,7 @@
-# Copyright 2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"). You
-# may not use this file except in compliance with the License. A copy of
-# the License is located at
-#
-#     http://aws.amazon.com/apache2.0/
-#
-# or in the "license" file accompanying this file. This file is
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
-
-
 module Aws
   module SessionStore
     module DynamoDB
-      VERSION = "1.0.0"
+      VERSION = "2.0.0"
```

Comment:
Load this from the file to reduce duplication here.

Reply:
In our other gems it's duplicated, I think in the SDK case it's automated to change.